askama/src/helpers.rs

```diff
@@ -267,9 +268,13 @@ impl<L: fmt::Display, R: fmt::Display> fmt::Display for Concat<L, R> {
 
 impl<L: FastWritable, R: FastWritable> FastWritable for Concat<L, R> {
     #[inline]
-    fn write_into<W: fmt::Write + ?Sized>(&self, dest: &mut W) -> crate::Result<()> {
-        self.0.write_into(dest)?;
-        self.1.write_into(dest)
+    fn write_into<W: fmt::Write + ?Sized>(
+        &self,
+        dest: &mut W,
+        values: &dyn Values,
```

Comment:
Shouldn't we have a `write_into_with_values` instead?

Reply:
Puh, I don't know. It would not be useful for the trait to have an implementation without `values`, so I think using this shorter name is fine.

Reply:
No I meant: is there a case where there is no `values` and therefore we could have a method without this argument like we do for the other trait?

Reply:
And now that I read more of the PR, the question kinda answers itself: it's needed because we always pass it to filters.